### PR TITLE
Custom editors for Swivel, Tumble

### PR DESCRIPTION
**Pull request by mjavaly**
Also renamed Swivel and Tumble, and changed their default prompts.

### CODE REVIEW

File: Project/Assets/Prairie/Framework/Script/Interaction/Swivel.cs [hunk 1]
```diff
@@ -65,7 +65,7 @@ protected void Rotate()
 
 	override public string defaultPrompt {
 		get {
-			return "Open Door";
+			return "Click to Open/Close Door";
```
Comment (grahamearley) [inline Project/Assets/Prairie/Framework/Script/Interaction/Swivel.cs]: I think we're getting rid of "Click to" in our prompts. The other default prompts just have the action. Could you change this to "Open/Close door"?

File: Project/Assets/Prairie/Framework/Script/Interaction/Tumble.cs [hunk 2]
```diff
@@ -64,7 +64,7 @@ protected void Update()
 
 	override public string defaultPrompt {
 		get {
-			return "Pick up Object";
+			return "Click to Pick Up Object";
```
Comment (grahamearley) [inline Project/Assets/Prairie/Framework/Script/Interaction/Tumble.cs]: Same here -- no "click to"

**Review by grahamearley — CHANGES_REQUESTED** [review]
Looking good. I made two minor change requests (removing "Click to" from your default prompts)

**Review by grahamearley — APPROVED** [review]
🛀 where's the like button???

**Review by shoemakerc — APPROVED** [review]
Great job!

**Pull request merged**